go/oasis-node/cmd/debug/txsource/workload/parallel.go

```diff
@@ -51,23 +60,31 @@ func (parallel) Run(gracefulExit context.Context, rng *rand.Rand, conn *grpc.Cli
 	fee := transaction.Fee{
 		Gas: parallelTxGasAmount,
 	}
+	if err = fee.Amount.FromInt64(parallelTxGasAmount); err != nil {
+		return fmt.Errorf("Fee amount error: %w", err)
+	}
+
+	for i := uint64(1); ; i++ {
 
-	for {
 		errCh := make(chan error, parallelConcurency)
 		var wg sync.WaitGroup
 		wg.Add(parallelConcurency)
-
-		for i := 0; i < parallelConcurency; i++ {
+		for c := 0; c < parallelConcurency; c++ {
 			go func(txSigner signature.Signer, nonce uint64) {
 				defer wg.Done()
 
 				// Transfer tx.
 				transfer := staking.Transfer{
 					To: txSigner.Public(),
 				}
-				tx := staking.NewTransferTx(nonce, &fee, &transfer)
+				if err = transfer.Tokens.FromInt64(parallelTxTransferAmount); err != nil {
```

Comment:
did we used to not have an amount set? 😬

Reply:
yeah since all the accounts used in this worklaod had 0 balance before this PR :grin: